src/test/java/ioanadroflorea/tests/Tema6/DemoQA.java

```diff
@@ -0,0 +1,89 @@
+package ioanadroflorea.tests.Tema6;
+
+import ioanandroflorea.driver.BrowserManager;
+import ioanandroflorea.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.Set;
+
+public class DemoQA {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQAWindowPage();
+        clickNewTabButton();
+        clickNewWindow();
+
+        try {
+            clickNewWindowMessage();
+        } catch (NoSuchElementException e) {
+            FileUtils.takeScreenshot(driver, "window message");
+        } finally {
+            BrowserManager.closeBrowser(driver);
+        }
+
+        BrowserManager.closeBrowser(driver);
```

Comment:
You are already executing this line in finally clause so please remove it from here. Thanks!